W-17361464 postrel polish duke (#325)
mulesoft/docs-monitoring: modules/ROOT/pages/dashboard-custom-config-text.adoc

* Markdown: See xref:dashboard-config-ref.adoc#samples_markdown[Markdown Syntax Support] for Markdown examples. a|

> image::text-config-html.png["Text panel in a custom dashboard"]

Author: ```suggestion
image::text-config-html.png["A dropdown menu displays "html" for Mode and "markdown" for Content."]
```

> image::text-config-html.png["A dropdown menu displays "html" for Mode and "markdown" for Content."]

Reviewer: ```suggestion
image::text-config-html.png["A dropdown menu displays "html" for Mode and "markdown" for Content"]
```

Author: not a complete sentence

|===

== Markdown Example

As this Markdown example shows, you can see how your text will appear in the UI as you type.

.Markdown Example

> image::text-config.png["Markdown example"]

Author: ```suggestion
image::text-config.png["A text editor that uses Markdown to format text."]
```

> image::text-config.png["A text editor that uses Markdown to format text."]

Reviewer: ```suggestion
image::text-config.png["A text editor that uses Markdown to format text"]
```

Author: not a full sentence
